pkg/minishift/registration/utils.go

```diff
@@ -86,6 +91,10 @@ func UnregisterHostVM(host *host.Host, param *RegistrationParameters) error {
 		if err := registrator.Unregister(param); err != nil {
 			return err
 		}
+		minishiftConfig.InstanceConfig.IsRegister = false
```

Comment:
The method could return the bool which then gets written into persistent config.